common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -152,27 +157,45 @@ export const SolidButton = styled(BaseButton).attrs<SolidButtonProps>(
     }),
   })
 )<SolidButtonProps>`
-  background: ${props =>
-    props.theme.color(
-      props?.colors?.background || props.theme.buttonColors.default.background
-    )};
-  color: ${props =>
-    props.theme.color(
+  padding: ${({ size }) => getPadding(size)};
+
+  ${props => `
+    background: 
+      ${props.theme.color(
+        props?.colors?.background || props.theme.buttonColors.default.background
+      )};
+    color: ${props.theme.color(
       props?.colors?.text || props.theme.buttonColors.default.text
     )};
-  border: 2px solid
-    ${props =>
-      props.theme.color(
-        props?.colors?.border || props.theme.buttonColors.default.border
-      )};
+  `}
 
-  padding: ${({ size }) => getPadding(size)};
+  ${props => {
+    if (props.isNewStyle) {
+      const { r, g, b } = hexToRgb(
```

Comment:
I read about [relative colour syntax](https://css-tricks.com/new-css-color-features-preview/#aa-the-relative-color-syntax-is-super-useful) last week. You can convert hex to hsl on the fly, which then lets you adjust alpha, e.g.

```
background: hsl(from `props.theme.color(
          props.colors?.background ||
            props.theme.buttonColors.default.background
        )` h s l / 0.8);
```

Probably makes sense to keep it as you have it here already though, but thought I'd mention it

Reply:
No this is brilliant! I don't think it's supported yet though, I've tried without success and struggle to find more info online 🤔 I see it on [w3.org](https://www.w3.org/TR/css-color-5/#:~:text=The%20new%20relative%20color%20syntax,possibly%20modified%20with%20math%20functions).) but their examples 404 🧐